app/pages/reservation/reservation-information/InternalReservationForm.js

```diff
@@ -0,0 +1,149 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { Component } from 'react';
+import { Field, reduxForm } from 'redux-form';
+import Form from 'react-bootstrap/lib/Form';
+import Row from 'react-bootstrap/lib/Row';
+import Col from 'react-bootstrap/lib/Col';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import injectT from '../../../i18n/injectT';
+import FormTypes from '../../../constants/FormTypes';
+
+export const maxLengths = {
+  internalReservationComments: 1500
+};
+
+function mapStateToProps(state) {
+  return { stateForm: state.form };
+}
+
+class UnconnectedInternalReservationForm extends Component {
+  state = {
+    internalReservationDefaultChecked: true,
+    markAsClosedDefaultChecked: false
+  }
+
+  componentDidMount() {
+    this.props.dispatch(this.internalReservationFormInitialValue());
+  }
+
+  internalReservationFormInitialValue() {
```

Comment:
Any reason for this?

Reply:
Internal reservation checkbox must be checked by default.
If I remove dispatch it doesn't send internal_reservation field and value
unless I first uncheck checkbox and then check again it.

Example payloads

```
{
  begin: "2019-11-02T11:00:00.000Z"
  comments: "Dispatch REMOVED, internal_reservation is missing!"
  end: "2019-11-02T12:00:00.000Z"
  resource: "av4pn7vgewja"
}
```

```
{
  begin: "2019-11-02T07:30:00.000Z"
  comments: "Dispatch REMOVED, uncheck checkbox and then check again it"
  end: "2019-11-02T08:00:00.000Z"
  internal_reservation: true
  resource: "av4pn7vgewja"
}
```

```
{
  begin: "2019-11-02T08:00:00.000Z"
  comments: "Dispatch ON"
  end: "2019-11-02T08:30:00.000Z"
  internal_reservation: true
  resource: "av4pn7vgewja"
}
```